### PR TITLE
Docker docker docker

### PR DESCRIPTION
**Pull request by alexjoybc**
# Description

This PR includes the following proposed change(s):

- adding dockerfile for pci gateway

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by akroon3r** [comment]
should we be adding a code climate git action on this?

File: .github/workflows/docker-build.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: Docker Build
+
+on:
+  pull_request:
+    branches: [master]
+  push:
```
Comment (bashbang) [inline .github/workflows/docker-build.yaml:6]: Is there a master branch?

**Review by bashbang — APPROVED** [review]

File: .github/workflows/docker-build.yaml (hunk 1 above)
Comment (alexjoybc) [reply .github/workflows/docker-build.yaml:6]: yup, should we switch to main ?

Comment (alexjoybc) [reply .github/workflows/docker-build.yaml:6]: no there is no lol

**Review by TayGov — APPROVED** [review]

**Pull request merged**